mw_url_rewrite.go

```diff
@@ -239,6 +239,14 @@ func valToStr(v interface{}) string {
 			}
 			i++
 		}
+	case []interface{}:
+		var tmpSlice []string
```

Comment:
would be better to do `tmpSlice := make([]string, 0, len(x))` to avoid slice re-allocations